.github/workflows/tests.yml

```
@@ -10,37 +10,39 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

So that it is pinned/reproducible?

---

Pinned to `ubuntu-24.04`,